pkg/console/dashboard_panels.go

```diff
@@ -213,9 +213,15 @@ func syncManagementURL(ctx context.Context, g *gocui.Gui) {
 
 func doSyncManagementURL(g *gocui.Gui) {
 	managementURL := "Unavailable"
-	if managementIP := getVIP(); managementIP != "" {
+	managementIP := getVIP()
+	if managementIP != "" {
 		managementURL = fmt.Sprintf("https://%s", managementIP)
+	} else {
+		if managementIP = getFirstReadyMasterIP(); managementIP != "" {
+			managementURL = fmt.Sprintf("https://%s:%s", managementIP, harvesterNodePort)
+		}
```

Comment:
On second thought, I think we should get the VIP from the vip configmap. The service is not ready until harvester is up and running and it could take some time. The console probably will show "Unavailable" => "https://some-node-ip:30443" => "https://vip". The switchover seems bad to users.